server/src/controllers/result.controller.js

```diff
@@ -1,13 +1,22 @@
 // Custom Modules
-//const response = require("../utils/response");
-//const CustomError = require("../utils/custom-error");
-//const DatabaseConnection = require("../db/database.helper");
+const response = require("../utils/response");
+const CustomError = require("../utils/custom-error");
+const DatabaseConnection = require("../db/database.helper");
 
-//const ResultRepo = new DatabaseConnection("001test_result");
+const ResultRepo = new DatabaseConnection("001test_result");
```

Comment:
this should be the games collection
since results are now fetched from games @fabianuzukwu

Reply:
that is ``new DatabaseConnection("001test_game");``